app/assets/javascripts/account/directives/back-to-application-link.html.slim

```diff
@@ -1,5 +1,5 @@
 .back-link-container.small-only-text-center ng-if="rememberedShortFormState"
-  a.back-link ui-sref="{{rememberedShortFormState}}"
+  a.back-link(ui-sref="{{rememberedShortFormState}}" ng-click="logBackLinkClick()")
```

Comment:
thought: I wonder about how useful this will be. 
- create-account: the button is clicked when the applicant starts creating an account, but changes their mind and goes back to the application
- sign-in: this now defaults to the React version of the page
- forgot-password: the button is clicked when the button to reset their password, then click back to the application

Reply:
@jimlin-sfgov Tenley can distinguish on her end where the user is coming from because the data sent to google analytics includes the origin url cc: @chadbrokaw